src/client/src/ui/blotter/model/blotterModel.js

```diff
@@ -41,6 +41,8 @@ export default class BlotterModel extends ModelBase {
     _log.info(`Blotter starting`);
     this._subscribeToConnectionStatus();
     this._regionManagerHelper.addToRegion();
+
+    setTimeout(this.subscribeToOpenFinEvents.bind(this), 1000);
```

Comment:
@KeithWoods do we have an event that is dispatched when the DOM is ready? OpenFin has an internal listener for this even when it sets up the desktop API. the 'init' event is dispatched too early, so I can't use that one.